tests/conftest.py

```diff
@@ -1134,6 +1131,23 @@ def get_state_db(self) -> DVSDatabase:
 
         return self.state_db
 
+    def get_fvs_dict(self, fvs):
+        fvs_dict = {}
+        for fv in fvs:
+            fvs_dict.update({fv[0]:fv[1]})
+        return fvs_dict
+
+    def change_port_breakout_mode(self, intf_name, target_mode, options=""):
+        cmd = "config interface breakout %s %s -y %s"%(intf_name, target_mode, options)
```

Comment:
```suggestion
        cmd = f"config interface breakout {intf_name} {target_mode} -y {options}"
```
f-string is preferred for python3